crates/sui-core/src/consensus_handler.rs

```diff
@@ -63,58 +63,74 @@ impl ConsensusHandler {
 impl ExecutionState for ConsensusHandler {
     /// This function will be called by Narwhal, after Narwhal sequenced this certificate.
     #[instrument(level = "trace", skip_all)]
-    async fn handle_consensus_transaction(
+    async fn handle_consensus_output(
         &self,
         // TODO [2533]: use this once integrating Narwhal reconfiguration
-        consensus_output: &Arc<ConsensusOutput>,
-        consensus_index: ExecutionIndices,
-        serialized_transaction: Vec<u8>,
+        consensus_output: ConsensusOutput,
     ) {
-        let index = Self::update_hash(
-            &self.last_seen,
-            consensus_index.clone(),
-            &serialized_transaction,
-        );
-        let index = if let Some(index) = index {
-            index
-        } else {
-            debug!(
+        let mut sequenced_transactions = Vec::new();
```

Comment:
Do you mind adding few utilization timers here?(you can see examples how `utilization_timer` is used in authority.rs for example)

Specifically -
(a) consensus_task_utilization for overall duration of `handle_consensus_output` function.
(b) When we have sorting, can we move that to separate function and have utilization timer for measuring sorting specifically